docs/content/updates/2024-12-09.mdx

```diff
@@ -0,0 +1,287 @@
+---
+date: 9th December 2024
+title: AgDS Beta v1.25.0 release
+description: Improves the accessibility and documentation for many components, improves the performance of Box and Combobox, initial release of yourgov app, plus various other bug fixes and improvements.
```

Comment:
Should this include `FeatureLinkList`, given it's a component? And possibly, though less importantly, the new icons?

The other thing I noticed is `yourGov` is already available on `main`, right? Or a we calling that an incomplete version and it's now officially released?

Reply:
Ah yeah, goo callout on FeatureLinkList and icons. As for yourgov, perhaps I should say "new yourgov app"?